package.json

```diff
@@ -77,6 +77,7 @@
     "@commitlint/config-conventional": "17.2.0",
     "@mdx-js/react": "2.1.5",
     "@storybook/addon-a11y": "6.5.13",
+    "@storybook/addon-controls": "^6.5.14",
     "@storybook/addon-essentials": "6.5.13",
```

Comment:
The storybook addon-essentials includes Controls. Was there a reason why you had to install it separately?
https://www.npmjs.com/package/@storybook/addon-essentials

Reply:
Can you point me to an example where you've a control has been disabled for a single story? I wasn't able to find one. This is why I've installed this addon.

Reply:
This addon should be included already through the essentials, it's what gave us Controls at all in the first place. 

We don't have any examples of it being disabled for a single story like you are implementing but similarly, it has been used to disable certain controls for all of the stories such as this one in ToggleInputList:
`    selectedItems: {
      control: { disable: true }
    },
    `

Reply:
I don't want to disable controls for all stories. I want to disable controls for just some of the stories. I tried it without the addon and it didn't work. If you think it can be done in another way without the addon, please let me know what that way is.

Reply:
I just wasn't work at all, it wouldn't disable the controls. I've tried uninstalling it now, and it's working 🤷‍♀️